Cargo.toml

```diff
@@ -14,3 +14,16 @@ panic = "abort"
 
 [profile.release]
 panic = "abort"
+
+# We take our fork of Solana 1.7.8, which has one patch applied:
+# https://github.com/solana-labs/solana/pull/21415.
+# This patch enables testing more situations using solana-program-test.
+# We need to override the other Solana packages as well because they are interdependent,
+# and if we don’t override them, Cargo will include two incompatible versions of those
+# packages (one from crates.io and one from the fork).
+[patch.crates-io]
+solana-frozen-abi = { git = "https://github.com/ChorusOne/solana", branch = "program-test-178" }
```

Comment:
That's merged, and it seems rebased for >= 1.7.8 👀.

Reply:
:eyes: https://github.com/solana-labs/solana/pull/21418 targeting 1.8 though. We can remove the overrides here once we are able to upgrade.